CHANGELOG.md

```diff
@@ -2,6 +2,18 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
+## [9.0.0](https://github.com/Ninja-Squad/ngx-valdemort/compare/v8.1.0...v9.0.0) (2022-11-18)
+
+
+### ⚠ BREAKING CHANGES
+
+* `ngx-valdemort` is now based on Angular 15.x. If you want to use it with an older version of Angular, then stick to a previous version of `ngx-valdemort`.
+
+### Features
+
+* make all components and directives standalone ([b4f7312](https://github.com/Ninja-Squad/ngx-valdemort/commit/b4f7312ee4a5c5afa01fcc55d28b2e61ac20b8e6))
```

Comment:
Maybe make that statement a bit more pro-eminent? Mayeb with the example from the docs?

Reply:
I will add some notes not the changelog after the release.